doc/quantum_error_correction.rst

```diff
@@ -30,3 +31,20 @@ Quantum Error Correction
             $(".nav-item a:contains('Quantum Error Correction')").parent().addClass("active");
         });
     </script>
+
+
+Introduction
+------------
```

Comment:
This would be easier to read and edit if it uses line-breaks at e.g., 100 chars (as the rest of the docs).

Are there any images that would fit in here, so that it's not just a big block of text? Alternatively, divide it into sections with subheadings. That would make it easier and nicer to read.

Reply:
Thanks! We will incorporate these in a future round of revisions.